bedrock/security/templates/security/bug-bounty/faq-webapp.html

```diff
@@ -78,13 +78,11 @@ <h2 id="eligible-bugs">Eligible bugs</h2>
     <dt id="eligible-bugs">Which domains and web applications will be considered to be part of the bug bounty?</dt>
     <dd>
       <p>Only a limited subset of our websites are eligible for monetary
-        payouts. Please see our <a href="https://hackerone.com/mozilla_critical_services/policy_scopes">HackerOne Mozilla Critical Services</a>
-        and <a href="https://hackerone.com/mozilla_core_services/policy_scopes">HackerOne Mozilla Core Services</a>
-        policy pages for additional information. Note that some low severity
+        payouts. Please see our <a href="https://hackerone.com/mozilla/policy_scopes">HackerOne program</a>
```

Comment:
**question:** this URL gives me a 404, is that expected and you're waiting on a change going live?